docs/source/oci/gettingstarted.rst

```diff
@@ -63,6 +63,13 @@ Then run the following command:
 
 After the configuration is complete, Cloud Custodian will implicitly pick up your credentials when it runs.
 
+
+Instance Principal Authentication
+"""""""""""""""""""""""""""""""""
+
```

Comment:
we should probably document the environment variable here, with a see more details at the link.

Reply:
Updated